### ISSUE TITLE
No keyboard nav instructions for moving objects

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/fourier-making-waves#168]
**Test device**
Dell
**Operating System**
Win 10
**Browser**
Chrome
**Problem description**
For https://github.com/phetsims/qa/issues/701.
The keyboard nav dialog on the third screen does not discus dragging objects around. This means there are no instructions for the k1 and lambda 1 measuring tools. They are not complicated to figure out, but we may want to add instructions anyway.

**Visuals**
<img width="726" alt="moveobjects" src="https://user-images.githubusercontent.com/41024075/131547645-14cb7196-4e42-408e-bc96-a0bc2e8560b2.png">

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Fourier: Making Waves‬
URL: https://phet-dev.colorado.edu/html/fourier-making-waves/1.0.0-dev.47/phet/fourier-making-waves_en_phet.html
Version: 1.0.0-dev.47 2021-08-29 19:29:57 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.159 Safari/537.36
Language: en-US
Window: 1280x657
Pixel Ratio: 1.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by pixelzoom** [comment]
That's also true of the first (**Discrete**) screen.  

@arouinfar please specify what (if anything) you'd like done here. If you want keyboard help, please provide a mockup of what that should look like.

**Comment by arouinfar** [comment]
For the Discrete and Wave Packet screens, add a "Measurement Tools" section directly below the "Slider Controls" section. The "Measurement Tools" title only appears on the Discrete screen, but I think it is appropriate for both screens. If you disagree, @pixelzoom , we can instead use "Component Spacing Tools" as the title on the Wave Packet Screen. 

We can model these sections after the "Move Book" section of Friction. 

![image](https://user-images.githubusercontent.com/8419308/131549259-dc676659-7f77-47be-81a6-0b1ff80d9ef0.png)

They keys will be exactly the same, but the text should read:

**Measurement Tools**
Move tool [arrows] or [WASD]
Move slower [Shift] + [arrows] or [Shift] + [WASD].

**Comment by pixelzoom** [comment]
@jessegreenberg @zepumph presumably what @arouinfar has specified in the comment about is the same for anything that has a KeyboardDragListener. 

For implementing:
> Move tool [arrows] or [WASD]

... I see `KeyboardHelpSelection.getGrabReleaseHelpSection`, but I can use it because it has language specific to "grab".  


For implementing 
> Move slower [Shift] + [arrows] or [Shift] + [WASD]

... it looks like Friction rolls its own help control in `MoveBookHelpSection`.

How do you you recommend that I proceed?  Do you want to provide reusable support for describing KeyboardDragListener keys?  Or should I continue with the "roll your own" pattern?

**Comment by pixelzoom** [comment]
On second thought... I'll start by cobbling together my own `MeasurementToolsKeyboardHelpSection`, and then have someone from the a11y team review it.

**Comment by zepumph** [comment]
The language (left column with English) tends to be sim specific enough that most of our factored out code has been in KeyboardHelpIconFactory with creating the right column Node with icons. I would recommend cobbling! Happy to review after the initial pass.

**Comment by pixelzoom** [comment]
Done in the above commit. Screenshots below is the same for **Discrete* and **Wave Packet** screens.

One deviations from @arouinfar's specification: the "Measurement Tools" section is ABOVE the "Slider Controls" section, because @terracoda said in a recent discussion that more sim-specific things come before more general things.

@arouinfar please review in master.
 
<img width="852" alt="screenshot_1228" src="https://user-images.githubusercontent.com/3046552/131707500-0c48ced8-bc8f-41a9-8223-eb7615c9c782.png">

**Comment by arouinfar** [comment]
> One deviations from @arouinfar's specification: the "Measurement Tools" section is ABOVE the "Slider Controls" section, because @terracoda said in a recent discussion that more sim-specific things come before more general things.

Good call! Looks good in master.

Not sure if the implementation needs further review, so back to @pixelzoom.

**Comment by pixelzoom** [comment]
I'm confident that the implementation is solid.  I reused everything that I could from `KeyboardHelpIconFactory` and `KeyboardHelpSection`.    So I'll close this issue.

@KatieWoe thanks for reporting. This is a nice improvement.
